### PR TITLE
item balance report

### PR DESCRIPTION
**Pull request by PawanMeh**
1. Simple Item Balance report to show minimal columns with stock quantity

item balance report

### CODE REVIEW

File: erpnext/stock/report/item_balance/item_balance.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+ "add_total_row": 0, 
+ "apply_user_permissions": 1, 
+ "creation": "2018-02-17 19:52:16.370979", 
+ "disabled": 0, 
+ "docstatus": 0, 
+ "doctype": "Report", 
+ "idx": 0, 
+ "is_standard": "Yes", 
+ "letter_head": "Nishal (Maharashtra)",
```
Comment (nabinhait) [inline erpnext/stock/report/item_balance/item_balance.json]: hard-coded letter head?

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/stock/report/item_balance/item_balance.json (hunk 1 above)
Comment (PawanMeh) [reply erpnext/stock/report/item_balance/item_balance.json]: removed with commit "358ffd7"

**Comment by nabinhait** [comment]
@PawanMeh 
The name of the reports (Item Balance / Stock balance) can be confusing for the user. Even I was confused while checking the PR, until I check the description of the PR. So, may be in similar line, the report can be called "Item Balance (Simple)" or "Stock Balance (Simple)"

**Review by maheshengg — APPROVED** [review]

**Comment by maheshengg** [comment]
@nabinhait @saurabh6790 

 Hi,  The report 'Item Balance (Simple)' hasn't been merged into the system, kindly reveiw and do the needful.

**Comment by maheshengg** [comment]
@PawanMeh How do I see this new report in my instance ??

**Comment by PawanMeh** [comment]
@maheshengg , this would be merged in master tomorrow end of day, so once it is available you can search by "Item Balance (Simple)" in search bar or reports and you should be able to view it.

**Comment by maheshengg** [comment]
@PawanMeh Need to know why I am unable to find ""Item Balance (Simple)" Report in the search bar. Help would be appreciated.

**Comment by PawanMeh** [comment]
@nabinhait , kindly confirm if it has been merged & updated in master branch or if there is an estimated timeline for it, you can let us know.

**Pull request merged**